### PR TITLE
feat: add moderated service link to welcome page

### PR DESCRIPTION
**Pull request by zbettenbuk**
image

### CODE REVIEW

File: react/features/welcome/components/AbstractWelcomePage.js [hunk 1]
```diff
@@ -269,16 +274,13 @@ export class AbstractWelcomePage extends Component<Props, *> {
...
- *     _room: string,
- *     _settings: Object
- * }}
+ * @returns {Props}
  */
 export function _mapStateToProps(state: Object) {
     return {
         _calendarEnabled: isCalendarEnabled(state),
         _enableInsecureRoomNameWarning: state['features/base/config'].enableInsecureRoomNameWarning || false,
+        _moderatedRoomServiceUrl: state['features/base/config'].moderatedRoomServiceUrl,
```
Comment (damencho) [inline react/features/welcome/components/AbstractWelcomePage.js:283]: Maybe add that to config.js commented with some explanation of how it is used.

Comment (zbettenbuk) [reply react/features/welcome/components/AbstractWelcomePage.js:283]: Thanks, added!

File: config.js [hunk 2]
```diff
@@ -571,7 +571,12 @@ var config = {
 
     // Allow all above example options to include a trailing comma and
     // prevent fear when commenting out the last value.
-    makeJsonParserHappy: 'even if last key had a trailing comma'
+    makeJsonParserHappy: 'even if last key had a trailing comma',
+
+    // The URL of the moderated rooms microservice, if available. If it
+    // is present, a link to the service will be rendered on the welcome page,
+    // otherwise the app doesn't render it.
+    moderatedRoomServiceUrl: 'https://moderated.jitsi.net'
```
Comment (saghul) [inline config.js:579]: Please don't put this after the end marker, `makeJsonParserHappy` should be the very last option here.

Also, don't leave it un-commented. The code should handle undefined.

File: react/features/base/config/configWhitelist.js [hunk 3]
```diff
@@ -123,6 +123,7 @@ export default [
     'liveStreamingEnabled',
     'localRecording',
     'minParticipants',
+    'moderatedRoomServiceUrl',
```
Comment (saghul) [inline react/features/base/config/configWhitelist.js:126]: I don't think it's a good idea to white-list it. Someone could use the external API to send you to some evil website.

File: react/features/welcome/components/AbstractWelcomePage.js [hunk 4]
```diff
@@ -25,6 +25,11 @@ type Props = {
      */
     _enableInsecureRoomNameWarning: boolean,
 
+    /**
+     * URL for the moderated rooms microservice, if available.
+     */
+    _moderatedRoomServiceUrl: string,
```
Comment (saghul) [inline react/features/welcome/components/AbstractWelcomePage.js:31]: `?string` ?

**Pull request merged**